www/assets/js/components/SearchPage.tsx

```diff
@@ -43,7 +44,12 @@ interface Result {
   _source: LearningResourceResult
 }
 
-export default function SearchPage() {
+type SearchPageProps = {
+  history: RouterHistory
+}
```

Comment:
Normally we would pass the history object via React Router's `useHistory` hook. But ocw doesn't use a router, so there's no point in adding the dependency.